applications/MeshingApplication/custom_processes/metrics_hessian_process.cpp

```diff
@@ -137,20 +140,19 @@ Vector ComputeHessianSolMetricProcess<TDim, TVarType>::ComputeHessianMetricTenso
     const double ElementMaxSize // This way we can impose as maximum as the previous size if we desire
     )
 {        
+    // We first transform the Hessian into a matrix
+    const bounded_matrix<double, TDim, TDim> hessian_matrix = MetricsMathUtils<TDim>::VectorToTensor(Hessian);
+    
     // Calculating Metric parameters
-    const double c_epslilon = mMeshConstant/mInterpError;
+    const double interpolation_error = mEstimateInterpError ? 2.0/9.0 * MathUtils<double>::Max(ElementMaxSize, ElementMaxSize * norm_frobenius(hessian_matrix)) : mInterpError;
```

Comment:
i would expand the if to a fully fledged if...else...

Also what happens if the mInterpError is 0?

Reply:
I can add a check